tests/validation/tests/v3_api/test_run_sonobuoy.py

```diff
@@ -0,0 +1,50 @@
+import base64
+import os
+import time
+from .common import *
+
+RANCHER_SONOBUOY_VERSION = os.environ.get("RANCHER_SONOBUOY_VERSION", "0.18.2")
+RANCHER_K8S_VERSION = os.environ.get("RANCHER_K8S_VERSION", "v1.18.2")
+RANCHER_SONOBUOY_MODE = os.environ.get("RANCHER_SONOBUOY_MODE",
+                                       "certified-conformance")
+RANCHER_KUBECONFIG = os.environ.get("RANCHER_KUBECONFIG")
+
+
+def test_sonobuoy_results():
+    config = base64.b64decode(RANCHER_KUBECONFIG).decode("utf-8")
+    kubeconfig = "./kubeconfig"
+
+    with open(kubeconfig, 'w') as f:
+        f.write(config)
+
+    cmd = "sonobuoy run --mode={0} --kube-conformance-image-version={1} --kubeconfig={2}".format(RANCHER_SONOBUOY_MODE, RANCHER_K8S_VERSION, kubeconfig)
```

Comment:
Each of this commands perform a different action, could we wrap them around methods so the main test looks more as reading steps?
```python
def test_sonobuoy_results():
    run_sonobuouy_conformance()
    wait_for_sonobouy_results() # <- in this method we can call get_sonobuoy_status()
    retrieve_sonobuoy_results()
    unpack_sonobuoy_results()
    show_sonobuoy_results()
    delete_sonobuoy_from_cluster()
```